### DISCUSSION TITLE
cubase 13.040 crash avec déplacement de pistes

### DISCUSSION THREAD

**Discussion by danimusicos** [bjoluc/cubase-mcu-midiremote#42] [General]
Bonjour, qui a un crash avec cubase 14.0.20 au moment de déplacer une piste cubase se plante. Le problème existait sur la version 13.0.30 avec la version antérieure aux 2 nouvelles du script mais de façon aléatoire.

**Comment by danimusicos** [comment]
idem cubase 14. 0.20

**Reply by limageurpublic** [reply]
Hello,
je n'ai pas constaté ces soucis avec cubase 12, 13 et 14....
à suivre .

**Reply by danimusicos** [reply]
salut, problème réglé avec cubase 14. mais sans savoir pourquoi çà le faisait de façon aléatoire avec mes deux xtouch
